Gemfile

```diff
@@ -6,3 +6,6 @@ if RUBY_VERSION < "1.9" && !%w{jruby rbx}.include?(ruby_engine)
   # Built and installed via ext/mkrf_conf.rb
   gem 'system_timer', '~> 1.2'
 end
+
+# fixes 2.3.0 ffi bundle error
+gem 'ffi', '~> 1.0.11'
```

Comment:
This appears to be necessary for bundling to work on ruby 2.3.0. I had already invested a fair amount of time before finding this solution so I didn't try variations. `~> 1.0` would allow the highest version currently available, which didn't work, so I think we should stick with this for now.